src/ert/config/queue_config.py

```diff
@@ -342,14 +377,24 @@ def throw_error_or_warning(
 
 
 def _validate_queue_driver_settings(
-    queue_system_options: List[Tuple[str, str]], queue_type: str, throw_error: bool
+    queue_system_options: List[Tuple[str, str]],
+    queue_type: str,
+    throw_error: bool,
+    realization_memory: int,
```

Comment:
Do we need `realization_memory` as a separate parameter, or can we only input queue_system_options and parse it here instead/also?

Reply:
I think it is okay for this private function, as realization_memory is not part of the tuple of strings constituting the queue_system_options. But I will reorder the arguments, making realization_memory the second arg.

Maybe we can do something about it later when refactoring queue_system_options from a list of tuples to e.g. a dataclass.